docs/on.md

```diff
@@ -33,7 +33,7 @@ on(button, 'click', function (event) {
 });
 
 ```
-## .once.
+## 'once'
```

Comment:
Should these be backticks?

Reply:
Maybe, but they are not for the other items in this readme...

Reply:
Ah, I see that was an oversight in my previous commit. They should all be backticks in this file.